### ISSUE TITLE
GPU Governor Tuneable

### ISSUE THREAD

**Issue by Oggie7797** [Grarak/KernelAdiutor#212]
Simple GPU gov allows the user to tune it to be more battery/performance based, kernel adiutor is missing this functionality. Other than that I really hate opening an "issue" when it's the best kernel manager out there!

**Comment by Grarak** [comment]
Simple GPU is already there?

**Comment by Oggie7797** [comment]
Yes my kernel includes it and I'm able to select and use it in kernel adiutor but I can't tweak any values like you're supposed to be able to and how you can with CPU governors.

**Comment by ShinySide** [comment]
Do you mean Simple GPU Algorithm which allows to select parameters?
